.travis/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+set -x
+
+if [ "$TRAVIS_OS_NAME" == "osx" ]; then
+    source .venv/bin/activate
+fi
+
+tox
```

Comment:
See the other file. Tox creates a virtual env on its own.

Reply:
I will remove the activation of the venv